foundry/game/gfx/GraphicsSet.py

```diff
@@ -55,8 +56,12 @@ class GraphicsSet:
 
     def __init__(self, graphic_set_number):
         if not self.GRAPHIC_SET_BG_PAGE_1:
-            self.GRAPHIC_SET_BG_PAGE_1 = ROM().bulk_read(BG_PAGE_COUNT, Level_BG_Pages1)
-            self.GRAPHIC_SET_BG_PAGE_2 = ROM().bulk_read(BG_PAGE_COUNT, Level_BG_Pages2)
+            self.GRAPHIC_SET_BG_PAGE_1 = self._heuristic_bg_pages(
+                unhexlify(b"0008101c0c58585c5830346e18381c242c5c586c683428"), Level_BG_Pages1
```

Comment:
This is some data and some code bytes? Maybe having those as a constant with some small doc string would be a bit better.

Reply:
Resolved by c45101a